data/BGC0001033.json

```diff
@@ -73,8 +82,7 @@
         "ncbi_tax_id": "697284",
         "organism_name": "Paenibacillus larvae subsp. larvae DSM 25430",
         "publications": [
-            "pubmed:25044543",
-            "pubmed:2508017"
```

Comment:
This should be 25080172

Reply:
Fixed with a new commit message/changelog